### PR TITLE
(#904) CollectionEnvelope.equals compare objects of Collection interface

### PR DESCRIPTION
**Pull request by scratchboom**

### CODE REVIEW

**Comment by 0crat** [comment]
This pull request [#1057](https://github.com/yegor256/cactoos/issues/1057) is assigned to @fabriciofx[/z](https://www.0crat.com/u/fabriciofx), here is [why](https://www.0crat.com/footprint/C63314D6Z/cdc84936-ac49-40b7-9917-d1ac07b7c5bb); the budget is 15 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§27](http://www.zerocracy.com/policy.html#27) and when you decide to accept the changes, inform @llorllale[/z](https://www.0crat.com/u/llorllale) (the architect) right in this ticket; if you decide that this PR should not be accepted ever, also inform the architect; this [blog post](http://www.yegor256.com/2015/02/09/serious-code-reviewer.html) will help you understand what is expected from a code reviewer; there will be no monetary reward for this job

File: src/test/java/org/cactoos/collection/CollectionEnvelopeTest.java [hunk 1]
```diff
@@ -144,6 +147,18 @@ public void differentHashCode() {
         );
     }
 
+    @Test
+    public void collectionEnvelopeShouldBeEqualToCollection() {
```
[the remark this answers is not in the document]
Comment (scratchboom) [reply src/test/java/org/cactoos/collection/CollectionEnvelopeTest.java]: @fabriciofx see new changes

**Comment by llorllale** [comment]
@fabriciofx ping

**Comment by llorllale** [comment]
@0crat refusepull/1057

**Comment by llorllale** [comment]
@0crat refuse

**Comment by 0crat** [comment]
> @0crat refuse ([here](https://github.com/yegor256/cactoos/issues/1057#issuecomment-468027523))

@llorllale The user @fabriciofx[/z](https://www.0crat.com/u/fabriciofx) resigned from [#1057](https://github.com/yegor256/cactoos/issues/1057), please stop working. Reason for job resignation: Order was cancelled

**Comment by llorllale** [comment]
@rultor merge

**Comment by 0crat** [comment]
This pull request [#1057](https://github.com/yegor256/cactoos/issues/1057) is assigned to @iprogrammerr[/z](https://www.0crat.com/u/iprogrammerr), here is [why](https://www.0crat.com/footprint/C63314D6Z/a9cf1a12-07fc-4197-a6f0-7deeaf39ca75); the budget is 15 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§27](http://www.zerocracy.com/policy.html#27) and when you decide to accept the changes, inform @llorllale[/z](https://www.0crat.com/u/llorllale) (the architect) right in this ticket; if you decide that this PR should not be accepted ever, also inform the architect; this [blog post](http://www.yegor256.com/2015/02/09/serious-code-reviewer.html) will help you understand what is expected from a code reviewer; there will be no monetary reward for this job

**Comment by llorllale** [comment]
@rultor release, tag is `0.40`

**Comment by 0crat** [comment]
Quality is low, no payment, see [§31](http://www.zerocracy.com/policy.html#31):

**Comment by 0crat** [comment]
Job was finished in 0 hours, bonus for fast delivery is possible (see [§36](http://www.zerocracy.com/policy.html#36))

**Comment by 0crat** [comment]
The job [#1057](https://github.com/yegor256/cactoos/issues/1057) is now out of scope

**Comment by 0crat** [comment]
Payment to `ARC` for a closed pull request, as in [§28](http://www.zerocracy.com/policy.html#28): +10 point(s) just awarded to @llorllale[/z](https://www.0crat.com/u/llorllale)

**Pull request merged**